layouts/shortcodes/code-webui-port-forward.html

```diff
@@ -1,2 +1,2 @@
 <pre><code>export NAMESPACE=istio-system
```

Comment:
Or you could delete this? How ever I think you can submit this PR with a better title, example https://github.com/kubeflow/manifests/pull/1176